### ISSUE TITLE
[CoE Starter Kit - QUESTION] Description of Admin | Sync Template v3 (Flow Action Details) may not match logic anymore

### ISSUE THREAD

**Issue by JohnMHearn** [microsoft/coe-starter-kit#7128]
### Does this question already exist in our backlog?

- [X] I have checked and confirm this is a new question.

### What is your question?

The description says that this flow gets action and trigger details for 'every individual flow in your tenant'.   After feature/issue #6576 and description in #6603 it seems this will only happen for those that are flagged as 'inventorymyflowactiondetails'.  The logic in the flow seems to confirm that.  Can this be confirmed, and the description be updated?   I would really like to use this valuable flow but have been reluctant because of the # of flows in our tenant. 

![image](https://github.com/microsoft/coe-starter-kit/assets/84465581/e83e3c1f-e5c5-4ed0-97a4-0ee5a0231966)

### What solution are you experiencing the issue with?

Core

### What solution version are you using?

October 4.16

### What app or flow are you having the issue with?

Admin | Sync Template v3 (Flow Action Details)

### What method are you using to get inventory and telemetry?

None

**Comment by Jenefer-Monroe** [comment]
That flag is a way to force the flow action details inventory for a particular flow. 

Any time a flow is modified, it goes through inventory, and as part of that inventory we mark the flow as needing its flow action details done as well.

**Comment by JohnMHearn** [comment]
We have too many flows in our tenant for this to run for all of our flows.  This is the reason why we couldn't turn this on in the first place.  We just want to gather the details on certain types of flows (for instance, those that use a certain connector).  I was hoping to limit the flow action details to just those by using this flag, but it seems that this flag/flow doesn't work like that.  In practice, introducing this flag only increases the number of flows that are interrogated for action details, doesn't help us to limit it.

**Comment by Jenefer-Monroe** [comment]
It wont be all flows in the inventory, it will be 

- all flows you manually flag to go through the process

plus

- all flows that go through inventory that day (because they were modified for example)

**Comment by JohnMHearn** [comment]
I think that assumes that the 'Flow Action Details' (FAD) flow has been on since October Core has been imported.  The FAD flow is the only one that sets that flag back to 'no'.  If you instead turn it on 1month after or intermittently, it will run for all the flows that have been inventoried since it last ran, which could be a lot.   I guess we need to decide when installing October, whether we will keep this flow always on or not.  If you turn it on months after, I guess it is recommended to first flip all flows that are 'yes' back to 'no' before you turn it on.  

Correct me if I'm wrong, but the old behavior of FAD was to run on all flows that are going through inventory that day (when a flow is added or modified), right?

**Comment by Jenefer-Monroe** [comment]
That's correct that was the old behavior. But it did not allow for the flow targeting feature which is needed for many governance scenarios. 

If you plan to turn this flow off and on so that you can use it for targeted activity, and don't want it to run for flows that have been modified since you last ran the FAD flow, you will need to go and clear that flag.

**Comment by JohnMHearn** [comment]
Thank you for that clear explanation.

**Comment by Jenefer-Monroe** [comment]
You are quite welcome!
